www/js/theme_editor.js

```diff
@@ -19,6 +20,83 @@ const themesWithCustomBg = themes
                             .filter(theme => theme.type === 'COLOR' || theme.type === 'SPECIAL')
                             .map(theme => theme.key);
 
+/*
+ * Helper Functions
+ */
+
+const uploadErrorNotification = (message) => {
+  new Noty({
+    type: 'error',
+    text: message,
+    timeout: 3000,
+    modal: true,
+  }).show();
+};
+
+const imageCheck = (filePath) => {
+  const acceptedExtensions = ['.png', '.PNG', '.jpg', '.JPG', '.jpeg', '.JPEG'];
+  const extension = path.extname(filePath);
+
+  return acceptedExtensions.includes(extension);
+};
+
+/*
+ * DOM Factories
+ */
+
+const recentSwatchFactory = backgroundPath =>
+  h(
+    'li.theme-instance.custom-bg',
+    {
+      style: {
+        'background-image': `url(${backgroundPath.replace(/(\s)/g, '\\ ')})`,
+      },
+      onclick: () => {
+        store.setUserPref('app.theme', themesWithCustomBg[0]);
+        store.setUserPref('app.themebg', {
+          type: 'custom',
+          url: backgroundPath.replace(/(\s)/g, '\\ '),
+        });
+        global.core.platformMethod('updateSettings');
+      },
+    },
+    h(
+      'button.delete-btn',
```

Comment:
can we fix the jumping on hover? 
![jump](https://user-images.githubusercontent.com/2982784/51695667-780be600-1fb8-11e9-9dcd-4ad536e01f0c.gif)

Reply:
@inderpreetsingh

Reply:
@tarunsingh5 that has already been fixed. 🙏🏽

Reply:
@tarunsingh5 that has already been fixed. 🙏🏽